hugegraph-api/src/main/java/com/baidu/hugegraph/serializer/JsonSerializer.java

```diff
@@ -85,7 +85,13 @@ public String writeList(String label, Collection<?> list) {
     private String writeIterator(String label, Iterator<?> iter,
                                  boolean paging) {
         // Early throw if needed
-        iter.hasNext();
+        try{
+            iter.hasNext();
+        }catch (NullPointerException e){
+            e.printStackTrace();
+        }catch (RuntimeException e){
+            e.printStackTrace();
```

Comment:
1. should align/format the code `{}` with space
2. maybe wrapper the exception rather than print stack trace is better
3. shall we ensure iter not null first ?